packages/hir/src/lowering/packages.candy

```diff
@@ -0,0 +1,22 @@
+use compiler_utils
+use incremental
+
+fun getDirectDependencies(context: QueryContext<List<CompilerError>>, package: Package): Set<Package> {
+  // Returns all direct dependencies of the given `package` as declared in the `.candyspec`.
+
+  todo("getDirectDependencies")
+}
+
+fun getTransitiveDependencies(context: QueryContext<List<CompilerError>>, package: Package): List<Package> {
+  // Returns all transitive dependencies of the given `package` in a topological order.
+  
+  todo("getTransitiveDependencies")
+}
+
+fun getTransitiveHullOfPackages(context: QueryContext<List<CompilerError>>, package: Package): List<Package> {
+  // Returns the transitive hull of packages of the given `package` in a topological order. This
+  // includes the `package` itself.
+
+  // TODO(marcelgarus): Implement properly.
+  return List.of1<Package>(package)
+}
```

Comment:
Isn't this like `getAllPackagesInCompilationOrder` from `compiler_utils/src/dependencies.candy`?